### PR TITLE
Disable assistive align

### PR DESCRIPTION
**Pull request by NikhilSuresh24**
kills assistive align when button 6 is pressed, and reruns it when button 6 is released.

### CODE REVIEW

File: src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java [hunk 1]
```diff
@@ -62,9 +63,13 @@ public void bindCommands() {
 		RobotMap.Component.driverXbox.dPad.right.whenPressed(new ChassisTurnAbsolute(RobotMap.Component.chassis, 90,
 			RobotMap.Component.navx, RobotMap.Component.chassisDriveMC));
 		RobotMap.Component.driverXbox.dPad.right.whenReleased(normalDrive);
+		// Inverted airplane-style analog control
+		new Climb(() -> Math.max(0, -RobotMap.Component.driverXbox.rightStick.getY())).start();
```
Comment (ajnadel) [inline src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java]: This duplicated line 74.

File: src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java [hunk 2]
```diff
@@ -62,9 +63,13 @@ public void bindCommands() {
...
 		RobotMap.Component.driverXbox.b.onlyWhileHeld(HumanInterfaceConfig.gearAlign);
 		RobotMap.Component.driverXbox.b.whenReleased(normalDrive);
-		RobotMap.Component.teensyStick.button1.whenPressed(normalDrive);
+		RobotMap.Component.teensyStick.getButton(0).whenPressed(normalDrive);
+		RobotMap.Component.teensyStick.getButton(5).whenPressed(new Kill(alignAssist));
```
Comment (ajnadel) [inline src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java]: This should be `new Cancel()` instead of Kill.

Comment (ajnadel) [reply src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java]: Actually, try using `onlyWhileReleased`. I think that's exactly what you're looking for.

**Review by ajnadel — CHANGES_REQUESTED** [review]

**Comment by ajnadel** [comment]
Also, you have merge conflicts :/

File: src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java (hunk 2 above)
Comment (NikhilSuresh24) [reply src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java]: it's going to be a switch though, right? So it shouldn't matter???

**Comment by c0d3rman** [comment]
Made obsolete by #69

**Pull request closed without merging**